CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Unreleased 
 - Alow users to filter get datasets by sources that they reference
 
```

Comment:
You should add this to `0.27.0` as you are releasing it :)

Reply:
Oh that's right, thanks!